File: src/sdk-reference/java/1/auth/delete-my-credentials/snippets/delete-my-credentials.java
try {
    kuzzle.getAuth().login("local", "{\"username\":\"foo\",\"password\":\"bar\"}");
    kuzzle.getAuth().deleteMyCredentials("local");
    System.out.println("Success");
} catch (KuzzleException e) {
    System.err.println(e.getMessage());
}

File: src/sdk-reference/js/6/auth/delete-my-credentials/snippets/delete-my-credentials.js
const credentials = { username: 'foo', password: 'bar' };

try {
  await kuzzle.auth.login('local', credentials);

  await kuzzle.auth.deleteMyCredentials('local');

  console.log('Success');
} catch (error) {
  console.error(error.message);
}
